doc/example/conversion_reaction.ipynb

```diff
@@ -157,7 +158,7 @@
     "\n",
     "# do the optimization\n",
     "result = optimize.minimize(\n",
-    "    problem=problem, optimizer=optimizer, n_starts=100, filename=None\n",
+    "    problem=problem, optimizer=optimizer, n_starts=10, filename=None\n",
```

Comment:
is this sufficient to get nice results always? There's no seed in this notebook. So it should be nice for any startpoints.

Reply:
conversion reaction looks good imo 👍🏼